resources/lib/modules/updates.py

```diff
@@ -464,6 +464,14 @@ def get_available_builds(self, shortname=None):
 
         If called without an argument, return a list of compatible builds with the running image.
         """
+
+        def pretty_filename(s):
+            """Make filenames prettier to users."""
+            s = self.lchop(s, f'{oe.DISTRIBUTION}-{oe.ARCHITECTURE}-')
+            s = self.rchop(s, '.tar')
+            s = self.rchop(s, '.img.gz')
+            return s
+
         channel = self.struct['update']['settings']['Channel']['value']
         matches = []
```

Comment:
It is unrelated to this PR. But in case that the update_json contains mixed `file` and `image`, the value does not get set to `[]` if the regex in line 522 matches once. Thus, shouldn't we move this `matches = []` to line 521?

Reply:
Yes, that should be sound. Thanks!